stablehlo/dialect/TypeInference.cpp

```diff
@@ -310,6 +310,41 @@ LogicalResult verifyReshapeOpQuantizationConstraints(
   return success();
 }
 
+LogicalResult verifyBroadcastInDimOpQuantConstraints(
+    std::optional<Location> location, Value operand, Value result,
+    ArrayRef<int64_t> broadcastDimensions) {
+  auto operandType = cast<RankedTensorType>(operand.getType());
+  auto resultType = cast<RankedTensorType>(result.getType());
+  auto resultQType = cast<quant::UniformQuantizedPerAxisType>(
+      getElementTypeOrSelf(result.getType()));
+  auto operandQType = cast<quant::UniformQuantizedPerAxisType>(
+      getElementTypeOrSelf(operand.getType()));
+  auto operandQDim = operandQType.getQuantizedDimension();
+  auto resultQDim = resultQType.getQuantizedDimension();
+  // broadcast_in_dimm_c6, dynamic_broadcast_in_dimm_c6
```

Comment:
dimm -> dim

Reply:
it came from https://en.wikipedia.org/wiki/DIMM ;) 

corrected, thanks.